src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Deprecation.php

```diff
@@ -61,10 +60,10 @@ public function __construct($message, array $trace, $file)
         $line = $trace[$i];
         $this->triggeringFile = $file;
         if (isset($line['object']) || isset($line['class'])) {
-            if (isset($line['class']) && 0 === strpos($line['class'], SymfonyTestsListenerFor::class)) {
-                set_error_handler(function () {});
-                $parsedMsg = unserialize($this->message);
-                restore_error_handler();
+            set_error_handler(function () {});
+            $parsedMsg = @unserialize($this->message);
```

Comment:
the `@ is not required anymore since the error handler ignores everything

Reply:
It's in master - that's why I added it back.

Reply:
I'll remove it here and submit a PR to clean up master.

Reply:
Created #38033 to address this in master.